code/src/Teletub.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import './Teletubbies.css'
+import img from './images/teletubbies.jpg'
+
+const Teletub = props => {
+    const {question, setTeletubbies, teletubbies} = props
+    return (
+        <div className="teletubbies-wrapper">
+            <h2 tabIndex="0">{question}</h2>
+            <img
+                tabIndex="0"
+                className="tt-image"
+                alt="visualization of a south park siter of teletubbies"
+                src={img}
+            />
+            <label htmlFor="teletub">Choose one of the four teletubbies</label>
```

Comment:
Chromevox screenreader skips to read this label when I tested. Maybe add tabindex here?

Reply:
I've added tabIndex to all labels now :)